### DISCUSSION TITLE
Merge Faces should always default to merge the person with no name into the person with a name.

### DISCUSSION THREAD

**Discussion by yurividal** [immich-app/immich#4772] [General]
When initiating a merge, immich should default to always merge the unamed person into the named person, if one of the 2 faces have a name. 

This is a simple, but very intuitive approach to the face merger

**Comment by johnpc** [comment]
Just want to second this idea, I just went through the process of merging a bunch of people and it seems to have almost always defaulted to the opposite way that I would prefer.

**Comment by davidpan** [comment]
You can try to name the unnamed face with the same name as the target face that needs to be merged, and you will be prompted whether to merge it into the target person's name.

The current face merging UI really needs improvement.
